requirements.txt

```diff
@@ -0,0 +1,74 @@
+#
```

Comment:
I don't think this file should be checked in to git - it should be generated as part of the build process really. We do this in the [Makefile in the devs.DC API](https://github.com/DemocracyClub/aggregator-api/blob/master/Makefile)